app/templates/index.html

```diff
@@ -0,0 +1,146 @@
+             691.971 241.379 693.974C240.911 695.978 239.977 697.843 238.654 699.419L246.234 705.783Z" fill="#EBEBFF"/>
+            <rect x="1112" y="102" width="204" height="47" transform="rotate(-30 1112 102)" fill="#EBEBFF"/>
+            <rect x="697.44" y="631" width="161.407" height="28.875" transform="rotate(23.34 697.44 631)" fill="#EBEBFF"/>
+            <rect width="56.0124" height="10.0203" transform="matrix(0.977401 -0.211395 -0.211395 -0.977401 659.253 142.572)" fill="#6070FF"/>
+            <rect x="0.601562" y="99.9232" width="108.669" height="19.4403" transform="rotate(-46.1415 0.601562 99.9232)" fill="#EBEBFF"/>
+            <circle cx="828.5" cy="662.5" r="23.5" fill="#6070FF"/>
+            <circle cx="685.5" cy="149.5" r="11.5" fill="#EBEBFF"/>
+            <circle cx="1177.5" cy="562.5" r="37" fill="white" stroke="#EBEBFF"/>
+            <circle r="19" transform="matrix(1 0 0 -1 950.5 53.5)" fill="white" stroke="#EBEBFF"/>
+            <circle cx="352.5" cy="53.5" r="7" fill="white" stroke="#EBEBFF"/>
+            <circle r="11" transform="matrix(1 0 0 -1 50.5 404.5)" fill="white" stroke="#EBEBFF"/>
+            <path d="M46.9842 73.0741C42.7936 82.8892 42.6736 93.967 46.6508 103.87C50.6279 113.774 
+            58.3763 121.692 68.1914 125.883C78.0065 130.073 89.0843 130.193 98.9878 126.216C108.891 
+            122.239 116.809 114.49 121 104.675L46.9842 73.0741Z" fill="#6070FF"/>
+            <path d="M221.155 650C220.821 654.881 222.439 659.695 225.654 663.383C228.87 667.071 233.418 
+            669.331 238.299 669.666C243.181 670 247.995 668.381 251.683 665.166C255.371 661.951 257.63 
+            657.403 257.965 652.521L221.155 650Z" fill="#6070FF"/>
+        </svg> 
+        
+        <!--Menu-->
+        <nav>
+            <div class="top-left">
+                <div class="menu"></div>
+                <div class="menu"></div>
+                <div class="menu"></div>
+            </div>
+            <div class="top-right">
+                <h2>My Logo</h2>
+            </div>
+        </nav>
+
+        <!-- Headline-->
+        <div class="centered">
+            <div class="primary-text">
+                <p>I’m Kennedy <br> Glad to see you!</p>
+            </div>
+            <div class="about-text">
+                <p>
+                    I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </p>
+            </div>
+            <div class="connect">
+                <p id="connect-text">LET’S CONNECT</p>
+            </div> 
+            <!--Social media--> 
+            <div class="social-icons">
+                <svg class="twitter" width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
- [ optional] you can see how to get direct icons [here ](https://www.javatpoint.com/how-to-add-social-media-icons-in-html#:~:text=social%20media%20icons.-,Step%201%3A%20Firstly%2C%20we%20have%20to%20type%20the%20Html%20code,sticky%20icons%20of%20social%20media.&text=Step%202%3A%20Now%2C%20we%20have%20to%20place%20the%20cursor%20just,then%20attach%20the%20CDN%20link.)instead of svg

Reply:
Thank you, let me work on the changes.